lib/substitutions.js

```diff
@@ -185,9 +185,78 @@ var htmlEscape = function(s) {
         .replace(/>/g, "&gt;");
 };
 
+
+// These functions are copied and modified from the Gitter AS
```

Comment:
This is kinda utterly gross but it looks like it works, surprised we can't use our own diffing solution. LGTM though.